blueprints/addon/files/README.md

```diff
@@ -4,18 +4,19 @@ This README outlines the details of collaborating on this Ember addon.
 
 ## Installation
 
-* `git clone` this repository
+* `git clone <repository-url>` this repository
+* `cd <%= addonName %>`
 * `npm install`
 * `bower install`
 
 ## Running
 
 * `ember serve`
-* Visit your app at http://localhost:4200.
+* Visit your app at [http://localhost:4200](http://localhost:4200).
```

Comment:
I don't have a strong preference, I just copied how it's written in the [app README](https://github.com/ember-cli/ember-cli/blame/master/blueprints/app/files/README.md#L26)